components/renku_data_services/message_queue/api.spec.yaml

```diff
@@ -0,0 +1,115 @@
+openapi: 3.0.2
+info:
+  title: Renku Data Services API
+  description: |
+    This service is the main backend for Renku. It provides information about users, projects,
+    cloud storage, access to compute resources and many other things.
+  version: v1
+servers:
+  - url: /api/data
+  - url: /ui-server/api/data
+paths:
+  /message_queue/reprovision:
+    post:
+      summary: Start a new reprovisioning
+      description: Only a single reprovisioning is active at any time
+      responses:
+        "201":
+          description: The reprovisioning is/will be started
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Reprovisioning"
+        "409":
+          description: A reprovisioning is already started
+        default:
+          $ref: "#/components/responses/Error"
+      tags:
+        - message_queue
+    get:
+      summary: Return status of reprovisioning
+      responses:
+        "200":
+          description: Status of reprovisioning if there's one in progress
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ReprovisioningStatus"
+        "404":
+          description: There's no active reprovisioning
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+        default:
+          $ref: "#/components/responses/Error"
+      tags:
+        - message_queue
+    delete:
+      summary: Stop an active reprovisioning
+      responses:
+        "204":
+          description: The reprovisioning was stopped or there was no one in progress
+        default:
+          $ref: "#/components/responses/Error"
+      tags:
+        - message_queue
+
+components:
+  schemas:
+    Reprovisioning:
+      description: A reprovisioning
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Ulid"
+        start_date:
+          description: The date and time the reprovisioning was started (in UTC and ISO-8601 format)
+          type: string
+          format: date-time
+          example: "2023-11-01T17:32:28Z"
+      required:
+        - id
+        - start_date
+      example:
+        - id: 01BX5ZZK2KAC4AV9WEV3EMM0S0
+          start_date: "2023-11-01T17:32:28Z"
```

Comment:
```suggestion
```
This is not needed. You will get the correct example from the definition of each field for free.

Or if you want to keep it then remove the `-` and indent 2 spaces to the left. Because the response is not a list.